### PR TITLE
chore(release): v0.8.1

### PR DESCRIPTION
**Pull request by jinlee93**
### Summary:
- patch eds for deps upgrades, esp style-dictionary

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
 
+## [0.8.1](https://github.com/chanzuckerberg/edu-design-system/compare/v0.6.0...v0.8.1) (2022-03-03)
+
+
+### Features
+
+* add ButtonGroup component ([#808](https://github.com/chanzuckerberg/edu-design-system/issues/808)) ([c413ac6](https://github.com/chanzuckerberg/edu-design-system/commit/c413ac674479919c2931e7dd03b5d5f08d18b43a))
+* add Dropdown component ([#835](https://github.com/chanzuckerberg/edu-design-system/issues/835)) ([acf54dc](https://github.com/chanzuckerberg/edu-design-system/commit/acf54dca68b784ca6718cc16927b7a97d55db6b8))
+* add DropdownButton ([#804](https://github.com/chanzuckerberg/edu-design-system/issues/804)) ([8b2d49f](https://github.com/chanzuckerberg/edu-design-system/commit/8b2d49f442e825e2e14ff8ad403f5a873edd5b8b))
+* add FormGroup component ([#809](https://github.com/chanzuckerberg/edu-design-system/issues/809)) ([3d48a2e](https://github.com/chanzuckerberg/edu-design-system/commit/3d48a2e2ea6d08fa88297e09ebfe3d5ee10b48f6))
```
Comment (dierat) [inline packages/components/CHANGELOG.md:14]: Is this stuff supposed to be in here? I'm confused 🤔

Comment (anniehu4) [reply packages/components/CHANGELOG.md:14]: hmm ya @jinlee93 are you on an old version of `main`? weird that it's comparing against v6

Comment (jinlee93) [reply packages/components/CHANGELOG.md:14]: hmm `main` was up to date, changed the `changelog` at root level, but didn't change for the package level, will update

Comment (jinlee93) [reply packages/components/CHANGELOG.md]: updated changelogs appropriately

**Review by dierat — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.8.1](https://github.com/chanzuckerberg/edu-design-system/compare/v0.8.0...v0.8.1) (2022-03-03)
```
Comment (ahuth) [inline CHANGELOG.md:6]: Wat. This is the first time  I noticed there's a top-level changelog.

Why would lerna/conventional-commits maintain this, when each package already has its own changelog? 🤔  Weird

**Review by ahuth — APPROVED** [review]

**Pull request merged**